src/senaite/core/profiles/default/workflows/senaite_sample_workflow/definition.xml

```diff
@@ -600,6 +604,7 @@
     <exit-transition transition_id="rollback_to_receive" />
     <exit-transition transition_id="detach" />
     <exit-transition transition_id="dispatch" />
+    <exit-transition transition_id="listing_multi_results"/>
```

Comment:
Do we really need this transition to be possible when the status of the sample is `to_be_verified`?. At this status is not possible to enter results, so I don't get the point.